This allows us to get some function from math.h that other platforms are
getting from ieeefp.h.
Easy to send upstream since it already has ifdef statement.

--- comm-esr31/mozilla/xpcom/ds/nsMathUtils.h.orig	2015-06-04 17:42:51.616409449 -0700
+++ comm-esr31/mozilla/xpcom/ds/nsMathUtils.h	2015-06-04 17:42:51.621339786 -0700
@@ -13,6 +13,7 @@
 #include <float.h>
 
 #ifdef SOLARIS
+#include <math.h>
 #include <ieeefp.h>
 #endif
 
